cmake-tool/helpers/simulation.cmake

```diff
@@ -118,7 +118,7 @@ function(GenerateSimulateScript)
     elseif(KernelPlatformRpi3 AND KernelSel4ArchAarch64)
         set(QemuBinaryMachine "qemu-system-aarch64")
         set(sim_serial_opt "-serial null -serial mon:stdio")
-        set(sim_machine "raspi3")
```

Comment:
Is there a version of qemu where `raspi3` isn't supported? On my version (5.2.0) it is supported:
```
$ qemu-system-aarch64 -machine help
Supported machines are:
...
raspi3               Raspberry Pi 3B (revision 1.2) (alias of raspi3b)
raspi3b              Raspberry Pi 3B (revision 1.2)
...
```

Reply:
Later versions do not support `raspi3`.

Reply:
See https://wiki.qemu.org/ChangeLog/6.2.